packages/mask/src/plugins/IdeaMarket/SNSAdaptor/AccountView.tsx

```diff
@@ -0,0 +1,231 @@
+                                                    {token.twitter ? (
+                                                        <Typography
+                                                            title={`${TWITTER_BASE_URL}/${token.twitter.username}`}
+                                                            className={classes.market}>
+                                                            <Link
+                                                                target="_blank"
+                                                                href={`${TWITTER_BASE_URL}/${token.twitter.username}`}>
+                                                                {TWITTER_BASE_URL}/{token.twitter.username}
+                                                            </Link>
+                                                        </Typography>
+                                                    ) : (
+                                                        <Typography title={token.name} className={classes.market}>
+                                                            <Link
+                                                                target="_blank"
+                                                                href={`${TWITTER_BASE_URL}/${token.name.slice(1)}`}>
+                                                                {TWITTER_BASE_URL}/{token.name.slice(1)}
+                                                            </Link>
+                                                        </Typography>
+                                                    )}
+                                                </Grid>
+                                            ) : (
+                                                <Grid item>
+                                                    <Typography title={token.name}>
+                                                        {urlWithoutProtocol(token.name)}
+                                                    </Typography>
+
+                                                    <Typography title={token.name} className={classes.market}>
+                                                        <Link target="_blank" href={token.name}>
+                                                            {token.name}
+                                                        </Link>
+                                                    </Typography>
+                                                </Grid>
+                                            )}
+                                        </div>
+                                    </TableCell>
+                                    <TableCell align="center">&#36;{tokenPrice.toFixed(2, 1)}</TableCell>
+                                    <TableCell align="center">{userTokenBalance.toFixed(2, 1)}</TableCell>
+                                    <TableCell align="center">&#36;{perceivedBalanceValue.toFixed(2, 1)}</TableCell>
```

Comment:
Just use the `dollar` symbol `$` for better readability.
```suggestion
                                    </TableCell>
                                    <TableCell align="center">${tokenPrice.toFixed(2, 1)}</TableCell>
                                    <TableCell align="center">{userTokenBalance.toFixed(2, 1)}</TableCell>
                                    <TableCell align="center">${perceivedBalanceValue.toFixed(2, 1)}</TableCell>
```